src/main.cpp

```diff
@@ -7306,7 +7306,7 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
 void AddPeek(std::string data)
 {
     return;
-    std::string buffer = RoundToString((double)GetAdjustedTime(),0) + ":" + data + "<CR>";
+    std::string buffer = std::to_string(GetAdjustedTime()) + ":" + data + "<CR>";
```

Comment:
this u should verify its invovled with peek in ProcessMessages() which im looking at converting sum. let me know